### PR TITLE
relisting at masternode online

### PR DESCRIPTION
**Pull request by hakonamatata**
Proposal to get relisted at masternode online

### CODE REVIEW

File: hakonamatata-relisting-at-masternodes-online.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: fr
+title: Pay for relisting/rebranding at masternodes.online
+author: 'JB'
+date: January 22, 2021
+amount: 500
```
Comment (justanwar) [inline hakonamatata-relisting-at-masternodes-online.md]: Amount needs to be in Firos

File: hakonamatata-relisting-at-masternodes-online.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+milestones:
+  - name: Pay for relisting/rebranding at https://masternodes.online/
+    funds: 500
```
Comment (justanwar) [inline hakonamatata-relisting-at-masternodes-online.md]: Here too

File: hakonamatata-relisting-at-masternodes-online.md (hunk 1 above)
Comment (hakonamatata) [reply hakonamatata-relisting-at-masternodes-online.md:6]: ok, so "500 FIRO", right?

**Comment by justanwar** [comment]
Excellent initiative by the community. Will merge to FR once minor issues are fixed.

**Comment by reubenyap** [comment]
This proposal is now funded. We will proceed to contact masternodes.online for relisting. Spoke to @hakonamatata who confirms he prefers team members to contact them.

**Comment by hakonamatata** [comment]
> 
> 
> This proposal is now funded. We will proceed to contact masternodes.online for relisting. Spoke to @hakonamatata who confirms he prefers team members to contact them.

Great, yes, it is probably better for a official team member to contact them.

**Comment by reubenyap** [comment]
I have paid the relisting fee. https://explorer.bitcoin.com/btc/tx/0b8fc398cb734740709d2e82a54861e9fdde75661c4ae718bc714db5f38036b9

**Comment by reubenyap** [comment]
The FIRO amount has been sent to me. https://explorer.firo.org/tx/4b48cfd908a59849bb0aa7c24ce40d4234c6c13e239c5da8026a065d8ef2a48f

**Comment by justanwar** [comment]
We are now listed on https://masternodes.online/currencies/FIRO/

Moving this to Completed

**Pull request merged**